src/sparseml/exporters/transforms/flatten_qparams.py

```diff
@@ -21,6 +23,8 @@
 
 __all__ = ["FlattenQParams"]
 
+_LOGGER = logging.getLogger(__name__)
```

Comment:
could add a shared logger in the base `Transform` class that prepends standard information about the transform like `self.__class__.__name__` to log calls